run.sh

```diff
@@ -0,0 +1,13 @@
+git clone https://github.com/appwrite/appwrite
+cd appwrite
+
+docker-compose up -d
+
+cd ~
+
+git clone https://github.com/appwrite/playground-for-php
+
+composer require 'appwrite/appwrite'
```

Comment:
Since you provide a composer.json, simply run `composer install` instead.

Reply:
I don't see a change here?

Reply:
Alternatively, download a fresh copy of the docker-compose.yml when executing this script.

But you still do a composer require rather than a simple composer install ;)